<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>canon</class>
 <widget class="QMainWindow" name="canon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 28, 47);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 28, 47);</string>
   </property>
   <widget class="QFrame" name="optionsMenu">
    <property name="geometry">
     <rect>
      <x>-43</x>
      <y>0</y>
      <width>231</width>
      <height>449</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(55, 66, 86);
border-radius: 50px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>211</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>75</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="manual_menu">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(23, 28, 47);
font: 700 9pt &quot;Tahoma&quot;;
color: rgb(249, 255, 239);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>MANUAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="automatic_menu">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(23, 28, 47);
font: 700 9pt &quot;Tahoma&quot;;
color: rgb(249, 255, 239);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>AUTOMATICO</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>automatic_menu</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>manual_menu</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="manualWitget" native="true">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>0</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(23, 28, 47), stop:1 rgb(15, 18, 30));


</string>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/cct/dragon_no_fondo.png);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>200</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 27, 46);
color: rgb(0, 0, 0);
font: 700 13pt &quot;Segoe UI&quot;;
border-radius: 20px
</string>
     </property>
     <property name="text">
      <string>CALCULATOR CANON</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="angulo_base">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Base:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Ángulo del cañón:</string>
     </property>
    </widget>
    <widget class="QSlider" name="angulo_canon">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="disparar_manual">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 16, 26);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(250, 231, 222);
border-raidus: 10px;</string>
     </property>
     <property name="text">
      <string>Disparar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recarga">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 16, 26);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(250, 231, 222);
</string>
     </property>
     <property name="text">
      <string>Recargar</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="autoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>0</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">


background-color: rgb(23, 28, 47);</string>
    </property>
    <widget class="QLabel" name="image2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/cct/dragon_no_fondo.png);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>200</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 27, 46);
color: rgb(0, 0, 0);
font: 700 13pt &quot;Segoe UI&quot;;
border-radius: 20px
</string>
     </property>
     <property name="text">
      <string>CALCULATOR CANON</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="barra_base_auto">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Base:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string> Distancia(cm):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disparar_auto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>340</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 16, 26);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(250, 231, 222);
border-raidus: 10px;</string>
     </property>
     <property name="text">
      <string>Disparar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color :rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="distancia_auto">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>130</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="ajustar_auto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 16, 26);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(250, 231, 222);
border-raidus: 10px;</string>
     </property>
     <property name="text">
      <string>Ajustar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inicion" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QLabel" name="info">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 238, 250);
font: 700 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>200</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 27, 46);
color: rgb(0, 0, 0);
font: 700 13pt &quot;Segoe UI&quot;;
border-radius: 20px
</string>
     </property>
     <property name="text">
      <string>CALCULATOR CANON</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Escoja un modo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>autoWidget</zorder>
   <zorder>manualWitget</zorder>
   <zorder>inicion</zorder>
   <zorder>optionsMenu</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>manual_menu</sender>
   <signal>clicked()</signal>
   <receiver>manualWitget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automatic_menu</sender>
   <signal>clicked()</signal>
   <receiver>manualWitget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angulo_base</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angulo_canon</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automatic_menu</sender>
   <signal>clicked()</signal>
   <receiver>autoWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manual_menu</sender>
   <signal>clicked()</signal>
   <receiver>autoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barra_base_auto</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manual_menu</sender>
   <signal>clicked()</signal>
   <receiver>inicion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automatic_menu</sender>
   <signal>clicked()</signal>
   <receiver>inicion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
